packages/frontend/.cct/utils/get-template-component-webpack-config.js

```diff
@@ -111,18 +104,26 @@ export const getTemplateComponentWebpackConfig = (
             // TODO: further globals might be passed by carrot-scripts??
             new webpack.DefinePlugin({
                 ...globals,
-                __DEV__: devMode ? JSON.stringify(true) : JSON.stringify(false),
+                ROOT_ID: JSON.stringify(POSTCSS_PREFIX),
+                __DEV__: JSON.stringify(!!devMode),
             }),
             new MiniCssExtractPlugin(),
             new webpack.container.ModuleFederationPlugin({
-                name: devMode ? `${type}/remoteEntry` : "remoteEntry",
-                library: {
-                    type: "window",
-                    name: uniqueName,
-                },
+                name: type,
+                filename: devMode ? `${type}/remoteEntry.js` : "remoteEntry.js",
```

Comment:
It's just a shorter way to specify the federated component's name (how it will be attached to the browser's window) vs the filename of the federated comnponent remote entry in the dist folder